broker/broker.go

```diff
@@ -308,8 +318,21 @@ func (b *CdnServiceBroker) checkDomain(domain, orgGUID string) error {
 }
 
 func (b *CdnServiceBroker) getHeaders(options Options) []string {
+	headers := make([]string, len(options.Headers))
+	copy(headers, options.Headers)
+	// Preserve original 'Host' header if forwarding to CloudFoundry's routing table.
```

Comment:
I don't quite understand this comment. At least it doesn't help me understand the conditional.

What conditions would `options.Origin` would not equal `b.settings.DefaultOrigin`?